fac29/NisaInvest-TFB-FE
# Dashboard home

## Description
I've created a home tab for the dashboard, which contains buttons that navigate to other tabs on the dashboard.

![image](https://github.com/user-attachments/assets/78603f16-b7a5-456e-a918-badd67017adc)

## Related Issue

## How Can This Be Tested?
Run the app and navigate to `/dashboard`. The default tab should be 'home'. Try clicking the 'profile' button and be amazed!

## Checklist:
